gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Package definitions
 GROUP=io.kategory
-VERSION_NAME=0.3.11-SNAPSHOT
+VERSION_NAME=0.3.11
```

Comment:
Shouldn't we bring it back to a SNAPSHOT version so it stays like that in master?